cocos/core/math/mat4.ts

```diff
@@ -218,10 +250,7 @@ export class Mat4 extends ValueType {
         let det = b00 * b11 - b01 * b10 + b02 * b09 + b03 * b08 - b04 * b07 + b05 * b06;
 
         if (det === 0) {
-            out.m00 = 0; out.m01 = 0; out.m02 = 0; out.m03 = 0;
-            out.m04 = 0; out.m05 = 0; out.m06 = 0; out.m07 = 0;
-            out.m08 = 0; out.m09 = 0; out.m10 = 0; out.m11 = 0;
-            out.m12 = 0; out.m13 = 0; out.m14 = 0; out.m15 = 0;
+            Mat4.zero(out);
             return out;
```

Comment:
Can just change to `return Mat4.zero(out);`

Reply:
Yes, I think we could. I will update it.

Reply:
Updated.